3scale-operator/tasks/main.yml

```diff
@@ -147,8 +238,15 @@
     state: absent
     api_version: v1
     kind: Secret
-    name: aws-auth
+    name: 3scaleportal
     namespace: "{{ target_namespace }}"
     wait: true
     wait_timeout: 60
     wait_sleep: 5
+- name: Clean up s3 bucket for 3Scale API Management
```

Comment:
this will give you an error because you need to empty contents of the bucket before you can remove the bucket

Reply:
Added the `force: true` to fix this, based on the logic of this module: 
https://github.com/ansible-collections/amazon.aws/blob/5c10d4d15b060a73ab25e5de811df63562c3273e/plugins/modules/s3_bucket.py#L1156